cloud/azure/sfx/integrations-azure.tf

```diff
@@ -12,7 +12,7 @@ resource "signalfx_azure_integration" "azure_integration" {
   app_id     = var.azure_sp_application_id
   secret_key = var.azure_sp_application_token
 
-  services = coalescelist(var.services, data.signalfx_azure_services.azure_services.services[*].name)
+  services = setsubtract(local.azure_services, var.excluded_services)
```

Comment:
I would applied the setsubstract only on the full service list from the datasource, it seems a bit weird but I suppose it should not be a problem while it is few probable a user provide a same service for both `services` and `excluded_services` variables.

Reply:
Hum, you're too optimistic with user usage of modules 😹 

This way we're sure excluded elements are always excluded :)